platform/golden/cfg/debian-11-preseed.cfg

```diff
@@ -0,0 +1,92 @@
+####################
+## General defaults
+
+d-i debian-installer/locale string en_US.UTF-8
+d-i keyboard-configuration/xkb-keymap select us
+d-i pkgsel/install-language-support boolean false
+
+####################
+## Network defaults
+
+d-i netcfg/choose_interface select auto
+d-i netcfg/get_hostname string golden
+d-i netcfg/get_domain string local
+
+
+###################
+## Mirror defaults
+
+d-i mirror/country string manual
+d-i mirror/http/hostname string http.us.debian.org
+d-i mirror/http/directory string /debian
+d-i mirror/http/proxy string
+d-i apt-setup/use_mirror boolean true
+
+#################
+## Time defaults
+
+d-i clock-setup/utc boolean true
+d-i time/zone string UTC
+d-i clock-setup/ntp boolean true
+
+##################
+## Login defaults
+
+d-i passwd/root-login boolean false
+d-i passwd/user-fullname string Artificial Wisdom
+d-i passwd/username string packer
+d-i passwd/user-password password packer
```

Comment:
this needs to be parameterized.

Reply:
done
Turns out you can pass any preseed 'd-i' path via kernel parameters.  Used this to add a password parameterization (from a random password created at build time.